RELEASE.md

```diff
@@ -97,7 +114,8 @@ Olivier Grisel, Patrick Christ, Povilas Liubauskas, Rainer Wasserfuhr,
 Romain Thouvenin, Sagan Bolliger, Sam Abrahams, Taehoon Kim, Timothy J Laurent,
 Vlad Zavidovych, Yangqing Jia, Yi-Lin Juang, Yuxin Wu, Zachary Lipton,
 Zero Chen, Alan Wu, @brchiu, @emmjaykay, @jalammar, @Mandar-Shinde,
-@nsipplswezey, @ninotoshi, @panmari, @prolearner and @rizzomichaelg.
+@nsipplswezey, @ninotoshi, @panmari, @prolearner, @rizzomichaelg, 
+and @danijar.
```

Comment:
Let's leave history alone and call out contributions to the bugfix release directly (as you already have) and make a proper list since 0.7 only for the next minor (0.8).